apisix/plugins/workflow.lua

```diff
@@ -82,6 +86,10 @@ local support_action = {
         handler  = exit,
         schema   = return_schema,
     },
+    ["limit-count"] = {
+        handler  = rate_limit,
+        schema   = limit_count.schema,
```

Comment:
We can't use group in the schema. As using group in the workflow will affect other limit-count configuration.

Reply:
fixed